install

```diff
@@ -296,6 +296,11 @@ Install the Linuxbrew dependencies:
 
 See http://brew.sh/linuxbrew/#dependencies for more information.
 
+Add to your #{Tty.white}.bashrc#{Tty.reset} or #{Tty.white}.zshrc#{Tty.reset}:
+  export PATH="$HOME/.linuxbrew/bin:$PATH"
+  export MANPATH="$HOME/.linuxbrew/share/man:$MANPATH"
+  export INFOPATH="$HOME/.linuxbrew/share/info:$INFOPATH"
+
```

Comment:
`shell_profile` won't work for the web site because it's Ruby code, but in shell we can say instead
```sh
echo 'export PATH="$HOME/.linuxbrew/bin:$PATH' >>~/.${SHELL#/bin/}rc
```